manage.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env python
+  
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.python.spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/manage.py
+++ b/manage.py
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-  
+
 import os
 import sys
 
```